### ISSUE TITLE
Add WER from most recent model

### ISSUE THREAD

**Issue by JRMeyer** [AASHISHAG/deepspeech-german#39]
Hi @AASHISHAG, and thanks for open-sourcing your work!

I'm wondering if you might have some performance stats (e.g. WER and CER) for more recent versions of the model (even just the most recent `v0.9.0`) would be good.

Thanks!

**Comment by AASHISHAG** [comment]
Hi @JRMeyer,

I'm glad to see your comment here. I have learned a lot from your Thesis, and blogs! Thank you so much. :)

Unfortunately, I couldn't record the WER for the latest model release. It was in the range of 10% to 20% on Mozilla and Tuda-De test set. But I don't remember it exactly, and I don't have the resources at the moment to run the setup.

However, I plan to train a new model with v0.10.0 and will provide all the stats whenever it's released from Coqui/DeepSpeech team.

**Comment by JRMeyer** [comment]
Glad to hear, @AASHISHAG :smile: 

I will be definitely looking forward to a new release... you should check out our chatroom on [Gitter](gitter.im/coqui-ai/STT) and come chat about the model, we can bounce ideas about hyperparams to train a stronger model :muscle:
